### PR TITLE
Update examples to target 316 boards

### PR DESCRIPTION
**Pull request by TDW89**
Addresses issue #40 
Update of the base examples to target xk-audio-316-mc

### CODE REVIEW

File: examples/app_spdif_rx_example/basic.xscope
Comment (xross) [inline examples/app_spdif_rx_example/basic.xscope:1]: is this required?

File: examples/app_spdif_rx_example/src/main.xc [hunk 1]
```diff
@@ -1,45 +1,72 @@
...
 #include <stddef.h>
 #include <spdif.h>
 #include <debug_print.h>
 
-port p_spdif_rx  = XS1_PORT_1F;
-clock audio_clk  = XS1_CLKBLK_1;
+on tile[0]: out             port    p_ctrl          = XS1_PORT_8D;
+on tile[0]: in              port    p_i2c_sda       = XS1_PORT_1M;
+on tile[0]: in              port    p_coax_rx       = XS1_PORT_1N;
+on tile[0]: in              port    p_adat_opt_rx   = XS1_PORT_1O;
```
Comment (xross) [inline examples/app_spdif_rx_example/src/main.xc:14]: p_adat_opt_rx would be clearer just as p_opt_rx (S/PDIF can use the opt jack)

File: examples/app_spdif_tx_example/src/main.xc [hunk 2]
```diff
@@ -1,25 +1,28 @@
...
+#include <spdif.h>
+#include <xassert.h>
 
-on tile[0] : buffered out port:32 p_spdif_tx    = XS1_PORT_1D;
-on tile[0] : in port p_mclk_in                  = XS1_PORT_1F;
-on tile[0] : clock clk_audio                    = XS1_CLKBLK_1;
+on tile[0]: out             port    p_ctrl          = XS1_PORT_8D;
+on tile[0]: in              port    p_i2c_sda       = XS1_PORT_1M;
+on tile[0]: in              port    p_coax_rx       = XS1_PORT_1N;
+on tile[0]: in              port    p_adat_opt_rx   = XS1_PORT_1O;
```
Comment (xross) [inline examples/app_spdif_tx_example/src/main.xc:12]: Same naming comment

**Review by xross — CHANGES_REQUESTED** [review]
Thanks for doing this. Just a couple of small comments.

File: examples/app_spdif_rx_example/basic.xscope
Comment (TDW89) [reply examples/app_spdif_rx_example/basic.xscope:1]: It's required for the app to use xscope, it compiles fine without it but you get an error when running with --xscope informing you that it's falling back on I/O via JTAG.

File: examples/app_spdif_tx_example/src/main.xc (hunk 2 above)
Comment (TDW89) [reply examples/app_spdif_tx_example/src/main.xc]: fixed

**Review by xross — APPROVED** [review]
As discussed early it would be good to move the AppPll settings out of here, but this is beyond the scope of this work.

**Pull request merged**